SUPPORT.md

```diff
@@ -3,3 +3,5 @@
 Thanks for trying out Velero! We welcome all feedback, please consider joining our mailing list: 
 
 - [Mailing List](https://groups.google.com/forum/#!forum/projectvelero)
+
+You can find details on the Velero maintainers' support process [here](https://velero.io/docs/master/support-process/).
```

Comment:
Terribly sorry that this thought just came to me now: It would be nice to point to the community page, which contains this and other info for contact: https://velero.io/community/, instead of the link to the google group.

Reply:
updated!